fed/mosaic-sns/src/test/java/org/eclipse/mosaic/fed/sns/model/AdhocTransmissionModelTest.java

```diff
@@ -57,7 +58,7 @@
     private final Map<String, SimulationNode> nodesRandomlyDistributed = new HashMap<>();
 
     @Rule
-    public GeoProjectionRule coordinateTransformationRule = new GeoProjectionRule(GeoPoint.lonLat(13.3856, 52.5415));
+    public GeoProjectionRule coordinateTransformationRule = new GeoProjectionRule(GeoPoint.lonLat(15.48, 47.02));
```

Comment:
The test only works, if the test takes places in the same UTM zone as the `GeoProjection` is configured with.